### ISSUE TITLE
[Feature]: ADDITION OF NATIVE AUTO COMPLETE CODE.

### ISSUE THREAD

**Issue by AllanFuego** [AndroidIDEOfficial/AndroidIDE#1105]
### Feature description

Hello Dear Android-IDE Team❤️
Hope all of you were fine🥰
My name is @allan_fuego and i am a game bug fixer + hunter.
I like to reverse games and make simple softwares with java,native and python.
I like your app very much as it can build apps in c++ 17.
But there is one problem i am really missing android studio native code auto complete feature in your app🥺🥺In java its there but in native there is not autocomplete function so can you plz add that feature as it will be really helpfull for jni app developers 🥺🙏🙏🙏Please IDE team focus on this feature its a very humble request i really need that🥺🥺and maybe many other devs too.

Lots of love to your team and respect to your work🥰❤️
Best of luck Guys❤️
Love from INDIA❤️🇮🇳🇮🇳

### What version of AndroidIDE you're using?

v2.4.0 (debug builds)

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by itsaky** [comment]
C++ LSP will be added eventually. But there are a lot of features/bugs to work on before we start working on this (Kotlin LSP as an example).

This is a duplicate of #1064.
